Library/Homebrew/dev-cmd/test.rb

```diff
@@ -92,6 +92,10 @@ def run
                 sandbox.allow_write_path(HOMEBREW_PREFIX/"var/homebrew/locks")
                 sandbox.allow_write_path(HOMEBREW_PREFIX/"var/log")
                 sandbox.allow_write_path(HOMEBREW_PREFIX/"var/run")
+                if f.class.offline_phase?(:test)
+                  sandbox.deny_all_network
+                  sandbox.allow_network(error_pipe, type: :literal)
```

Comment:
Feels like it might make sense to combine these calls given they are in the same order and the same every time?